core/templates/dev/head/pages/base.html

```diff
@@ -121,6 +121,8 @@
                       {% endblock local_top_nav_options %}
                     </div>
                   </div>
+                  {% block local_top_nav_options_short %}
+                  {% endblock local_top_nav_options_short %}
```

Comment:
Hi, we are trying to get rid of using Jinja on most of our pages and since the local_top_nav_options_short is only used on exploration editor we don't neet this Jinja construct.

Reply:
I have removed Jinga and imported the directive.